src/Pelagos/Entity/Account.php

```diff
@@ -192,7 +192,7 @@ public function __construct(Person $person = null, $userId = null, Password $pas
      */
     public function getId()
     {
-        return $this->getPerson();
+        return $this->getPerson()->getId();
```

Comment:
We should check if the object exists before we call a method on it

Reply:
That's right,  but now I don't remember why I put getId() here...